Bejebeje.DataAccess/Data/DataSeeder.cs

```diff
@@ -99,10 +100,10 @@ public void EnsureDataIsSeeded()
                 }
               },
               IsApproved = true,
-              ImageUrl = "https://placehold.it/100x100",
               UserId = userId,
               CreatedAt = DateTime.UtcNow,
-              Lyrics = new List<Lyric> { tnt }
+              Lyrics = new List<Lyric> { tnt },
+              Image = GetImage("acdc.jpg")
```

Comment:
Just a little note, this is why I prefer to always add that `,` on the end. Git prefers.